src/qt/transactionrecord.cpp

```diff
@@ -373,13 +372,13 @@ void TransactionRecord::loadUnlockColdStake(const CWallet* wallet, const CWallet
     record.involvesWatchAddress = false;
 
     // Get the p2cs
-    const CScript* p2csScript = nullptr;
+    CScript p2csScript;
     bool isSpendable = false;
 
     for (const auto &input : wtx.vin) {
         const CWalletTx* tx = wallet->GetWalletTx(input.prevout.hash);
         if (tx && tx->vout[input.prevout.n].scriptPubKey.IsPayToColdStaking()) {
-            p2csScript = &tx->vout[input.prevout.n].scriptPubKey;
+            p2csScript = tx->vout[input.prevout.n].scriptPubKey;
```

Comment:
(84a0785ddc9c887d159716339ac78d7d2d8380e1) why this change? using a pointer required one less copy of the script.

Reply:
to suppress a small warning on the `ExtractAddress` that is below. If `p2csScript` pointer is null (which should never happen), then undefined behavior.

Reply:
Yes, I thought so. But we could keep the performance advantage of using  a pointer, simply adding a condition to the last line:
```cpp
if (p2csScript)
    ExtractAddress(*p2csScript, false, record.address);
```

Reply:
done